src/main/java/net/openhft/chronicle/bytes/AbstractBytes.java

```diff
@@ -975,21 +934,27 @@ public int hashCode() {
 
     @Override
     public boolean equals(Object obj) {
-        if (!(obj instanceof BytesStore)) return false;
+        if (!(obj instanceof BytesStore)) {
+            return false;
+        }
         @NotNull BytesStore b2 = (BytesStore) obj;
         long remaining = readRemaining();
-        return b2.readRemaining() == remaining && equalsBytes(b2, remaining);
+        return (b2.readRemaining() == remaining) && equalsBytes(b2, remaining);
     }
 
     public boolean equalsBytes(@NotNull BytesStore b2, long remaining) {
         try {
             long i = 0;
-            for (; i < remaining - 7; i += 8)
-                if (readLong(readPosition() + i) != b2.readLong(b2.readPosition() + i))
+            for (; i < (remaining - 7); i += 8) {
+                if (readLong(readPosition() + i) != b2.readLong(b2.readPosition() + i)) {
                     return false;
-            for (; i < remaining; i++)
-                if (readByte(readPosition() + i) != b2.readByte(b2.readPosition() + i))
+                }
+            }
+            for (; i < remaining; i++) {
+                if (readByte(readPosition() + i) != b2.readByte(b2.readPosition() + i)) {
                     return false;
+                }
+            }
```

Comment:
This kind of reformatting changes makes it really hard to see what actually was changed. I already scrolled through over half of the diff and yet haven't seen the real change...
Also, introducing unnecessary `{ }` makes code cluttered. I guess it is your IDE doing this, I'd suggest you turn it off.

Reply:
Guys,
Can someone give me, it's InteliJ/eclipse code formtting preference file?

Reply:
I just use default Intellij's settings, no specific formatting used.

Reply:
I do not have IntelliJ, if is possible please export your preferences and sent them to me. I will try to find a way to import them in Eclipse.
Thanks

Reply:
Agree with Dmitry - it is very difficult to understand what change has been made. Suggest you turn off auto-formatting in your IDE